roles/validate/files/rules/required_rules/502_policy_vrf_lite_cross_reference.py

```diff
@@ -0,0 +1,257 @@
+'''
+Validation Rules scenarios:
+1.  Switches in the [insert vrf-lite switch model key structure here] should be defined in the vxlan.topology.switches
+2.  BGP and OSPF should be defined in two different policy.
+3.  route_reflector_client for AF IPv4 and IPv6 config should not be allowed.
+4.  Check if OSPF Authentication is enabled, key must be provided
+5.  Check AREA 0 is standard
+6.  Check static routes across Policies
+7.  Check ospf network type for Loopback
+'''
+
+
+class Rule:
```

Comment:
This looks much cleaner and readable now.  Let's include sample model data as a comment in the header for reference.  Makes it easier for anyone that needs to update this function/method going foward.

Reply:
Done. update data model